test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -44,6 +44,12 @@ Describe "Test-Connection" -tags "CI" {
         $targetAddress = "127.0.0.1"
         $targetAddressIPv6 = "::1"
         $UnreachableAddress = "10.11.12.13"
+        # this resolves to an actual IP rather than 127.0.0.1
```

Comment:
I don't understand why this commented out script is here.  Is this something we want to revert to, or avoid?